### ISSUE TITLE
Fix for error 9

### ISSUE THREAD

**Issue by arkulpaKonstantin** [below/WWDC22_CoreMediaIO#1]
Hey,

I just also implemented the sample code from the WWDC session and ran into the same issue as you did. That is actually how I found this repo.

So I investigated the issue further and first found [this page](https://developer.apple.com/documentation/systemextensions/ossystemextensionerror/code/validationfailed) where the error codes are described. With that information I then found the issue via [this post](https://stackoverflow.com/questions/65525013/system-extension-validation-fails). 

Long story short her is the fix that made it work for me:
Change the `CMIOExtensionMachServiceName` to `$(TeamIdentifierPrefix)com.vonbelow.sample.ExampleCam` in the extensions `info.plist` file

I hope this helps 🙂
